docs/baking.md

```diff
@@ -0,0 +1,175 @@
+<!--
+   - SPDX-FileCopyrightText: 2019 TQ Tezos <https://tqtezos.com/>
```

Comment:
Especially considering that this is a fresh file, let's start updating the copyright here.
```suggestion
   - SPDX-FileCopyrightText: 2021 TQ Tezos <https://tqtezos.com/>
```

Reply:
:see_no_evil: